application/controllers/digital_take_up.py

```diff
@@ -84,7 +131,27 @@ def channel_options(admin_client, uuid):
         if True in form.data.values():
```

Comment:
I think you could replace this with `if form.data.values()` because `bool(['thing'])` will evaluate to True.